airflow/utils/classes.py

```diff
@@ -0,0 +1,107 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from builtins import object
+
+import logging
+import signal
+
+
+class AirflowTaskTimeout(Exception):
```

Comment:
I was torn on this one since it is used right below in the timeout class. But I will move it to exceptions for consistency.